src/Components/addItem.js

```diff
@@ -1,139 +1,123 @@
-import React from 'react';
+import React, { useState } from 'react';
 import '../App.css';
 import { fb } from '../lib/firebase';
 import { getLocalToken } from '../lib/token.js';
 import { Alert } from 'react-bootstrap';
+import calculateEstimate from '../lib/estimates.js';
 
-class AddItem extends React.Component {
-  constructor() {
-    super();
-    this.state = {
-      itemName: '',
-      purchaseFrequency: null,
-      lastPurchasedDate: null,
-      errorMsg: null,
-    };
-  }
+function AddItem() {
+  const [itemName, setItemName] = useState('');
+  const [purchaseFrequency, setPurchaseFrequency] = useState(null);
+  const [lastPurchasedDate, setLastPurchasedDate] = useState(null);
+  const [errorMsg, setErrorMsg] = useState(null);
 
-  updateInput = e => {
-    this.setState({
-      [e.target.name]: e.target.value,
-    });
-  };
-
-  updateFrequency = e => {
-    this.setState({
-      purchaseFrequency: e.target.value,
-    });
-  };
+  const updateInput = e => setItemName(e.target.value);
+  const updateFrequency = e => setPurchaseFrequency(e.target.value);
+  const clearError = () => setErrorMsg(null);
 
-  
-  clearError = () => this.setState({ errorMsg: null });
-
- 
-  addItem = e => {
+  const addItem = e => {
     e.preventDefault();
     const db = fb.firestore();
     const collection = db.collection(getLocalToken());
 
-    var query = collection.where('itemName', '==', this.state.itemName);
+    var query = collection.where('itemName', '==', itemName);
 
     query
       .get()
       .then(querySnapshot => {
         if (querySnapshot.size) {
-          // console.log('this name is already in db:', this.state.itemName);
-          this.setState({ errorMsg: 'Item is already in list.' });
+          setErrorMsg('Item is already in list.');
         } else {
+          const date = new Date();
+          const nextPurchaseDate = new Date(date);
+          nextPurchaseDate.setDate(nextDate.getDate() + purchaseFrequency);
```

Comment:
Also, this is breaking the build - is `nextDate` what you want? or `date`?

cc @nabilkaz @Tanzi11